BinaryProtocol-8bit.md

```diff
@@ -94,60 +132,120 @@ Specifies the size of the word in the [`Payload`].
 
 #### HasTimestamp (1 bit)
 
-If this bit is set the Harp Message contains a timestamp. In this case the fields [`Seconds`] and [`Microseconds`] must be present in the message.
+If the Harp message contains a timestamp, this bit MUST be set. In this case the fields [`Seconds`](#seconds-4-bytes) and [`Microseconds`](#microseconds-2-bytes) MUST be included in the message.
 
 #### IsFloat (1 bit)
 
-This bit indicates whether the [`Payload`] represents fractional values. If the bit is not set, the payload contains integers.
+This bit indicates whether [`Payload`](#payload--bytes) encodes fractional values. If the bit is not set, the payload contains integers.
+
+> [!NOTE]
+>
+> The bit [`IsFloat`] MUST NOT be set with 8-bit or 16-bit sized payloads.
```

Comment:
Consider expanding this to make it clear that the protocol does not define "half-floats"?